### PR TITLE
Create bot_test.ttl

### PR DESCRIPTION
**Pull request by wterkaj**
Example showing how the BOT ontology can be used to describe the topology of a building.

### CODE REVIEW

File: bot/bot_test.ttl [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix dce: <http://purl.org/dc/elements/1.1/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix bot_test: <http://bot_test#> .
+@base <http://bot_test> .
```
Comment (maximelefrancois86) [inline bot/bot_test.ttl]: reminder for a next commit:

1. bot_test is not valid URL host name, should be ex. http://example.org/bot#
2. use a URI that ends with # or /, maybe the same as bot_test ?

File: bot/bot_test.ttl [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                   dce:description "Example showing how the BOT ontology can be used to describe the topology of a building.
+The building has four external walls and consists of 2 storeys. In the first storey there are rooms 101 and 102. In the second storey there is room 201.
+Room 101 has an entrance door, one window and a table.
+Room 102 has a window and a sofa.
+A door connects room 101 and room 102.
+A trapdoor connects room 102 and room 201." ;
+                   dce:creator "Walter Terkaj (walter.terkaj@itia.cnr.it)" ;
+                   cc:license <http://creativecommons.org/licenses/by/3.0/> ;
+                   owl:versionInfo "April 5th 2017"^^xsd:string ;
+                   dce:modified "April 5th 2017"^^xsd:string .
```
Comment (maximelefrancois86) [inline bot/bot_test.ttl]: use dcterms, and a xsd:dateTime ?

File: bot/bot_test.ttl [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+<http://bot_test> rdf:type owl:Ontology ;
+                   owl:imports <https://w3id.org/bot> ;
+                   dce:description "Example showing how the BOT ontology can be used to describe the topology of a building.
+The building has four external walls and consists of 2 storeys. In the first storey there are rooms 101 and 102. In the second storey there is room 201.
+Room 101 has an entrance door, one window and a table.
+Room 102 has a window and a sofa.
+A door connects room 101 and room 102.
+A trapdoor connects room 102 and room 201." ;
+                   dce:creator "Walter Terkaj (walter.terkaj@itia.cnr.it)" ;
```
Comment (maximelefrancois86) [inline bot/bot_test.ttl]: can use a full URI (some URL that points to a foaf description of you on your website ?)

File: bot/bot_test.ttl [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+@base <http://bot_test> .
+
+<http://bot_test> rdf:type owl:Ontology ;
+                   owl:imports <https://w3id.org/bot> ;
+                   dce:description "Example showing how the BOT ontology can be used to describe the topology of a building.
+The building has four external walls and consists of 2 storeys. In the first storey there are rooms 101 and 102. In the second storey there is room 201.
+Room 101 has an entrance door, one window and a table.
+Room 102 has a window and a sofa.
+A door connects room 101 and room 102.
+A trapdoor connects room 102 and room 201." ;
```
Comment (maximelefrancois86) [inline bot/bot_test.ttl]: linebreaks are authorized in turtle literals only if they are encloed in triple-double quotes

"this
is illegal literal"

"""this 
is legal literal"""

File: bot/bot_test.ttl [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix bot_test: <http://bot_test#> .
+@base <http://bot_test> .
+
+<http://bot_test> rdf:type owl:Ontology ;
+                   owl:imports <https://w3id.org/bot> ;
```
Comment (maximelefrancois86) [inline bot/bot_test.ttl]: owl prefix declaration is missing

File: bot/bot_test.ttl [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+                                  bot:Element .
+
+bot_test:window101 rdf:type owl:NamedIndividual ,
+                            bot:Element .
+
+bot_test:window102 rdf:type owl:NamedIndividual ,
+                            bot:Element .
+
+bot_test:window201 rdf:type owl:NamedIndividual ,
+                            bot:Element .
```
Comment (maximelefrancois86) [inline bot/bot_test.ttl:113]: rest of the doc "looks" good

**Review by maximelefrancois86 — CHANGES_REQUESTED** [review]

File: bot/bot_test.ttl (hunk 1 above)
Comment (wterkaj) [reply bot/bot_test.ttl]: 1. ok I will change the ontology URI in <http://example.org/bot_test>
2. ok I will change the namespace in <http://example.org/bot_test#>

File: bot/bot_test.ttl (hunk 5 above)
Comment (wterkaj) [reply bot/bot_test.ttl]: owl prefix is declared in line 6 of the current version

File: bot/bot_test.ttl (hunk 4 above)
Comment (wterkaj) [reply bot/bot_test.ttl]: it will be fixed accordingly

File: bot/bot_test.ttl (hunk 2 above)
Comment (wterkaj) [reply bot/bot_test.ttl]: dc:issued and dc:modified will be used with xsd:date

File: bot/bot_test.ttl (hunk 3 above)
Comment (wterkaj) [reply bot/bot_test.ttl]: I will add the url of personal foaf description.

**Pull request merged**